Modules/Sources/WordPressShared/Utility/Double+Stats.swift

```diff
@@ -158,32 +156,29 @@ extension Double {
 
 }
 
-extension NSNumber {
-    func abbreviatedString(forHeroNumber: Bool = false) -> String {
-        return self.doubleValue.abbreviatedString(forHeroNumber: forHeroNumber)
-    }
-
-    func percentageString() -> String {
-        return self.doubleValue.percentageString()
-    }
-}
-
-extension Float {
-    func abbreviatedString(forHeroNumber: Bool = false) -> String {
-        return Double(self).abbreviatedString(forHeroNumber: forHeroNumber)
+extension Int {
+    public func abbreviatedString(forHeroNumber: Bool = false) -> String {
+        Double(self).abbreviatedString(forHeroNumber: forHeroNumber)
     }
 
-    func percentageString() -> String {
-        return Double(self).percentageString()
+    public func percentageString() -> String {
+        Double(self).percentageString()
     }
 }
 
-extension Int {
-    func abbreviatedString(forHeroNumber: Bool = false) -> String {
-        return Double(self).abbreviatedString(forHeroNumber: forHeroNumber)
-    }
+private extension NumberFormatter {
+    static let statsPercentage: NumberFormatter = {
+        let formatter = NumberFormatter()
+        formatter.numberStyle = .percent
+        formatter.multiplier = 1
+        formatter.minimumFractionDigits = 0
+        formatter.maximumFractionDigits = 2
+        if let preferredLocaleIdentifier = Bundle.main.preferredLocalizations.first {
+            formatter.locale = Locale(identifier: preferredLocaleIdentifier)
+        } else {
+            formatter.locale = Locale.current
```

Comment:
I'm tempted to suggest injecting `Bundle.main` and `Locale.current` here, but it might result in a non-trivial sequence of call chain updates that might slow down the progress on moving this type.

It's just that seeing this library code accessing what to me feel like app-specific settings seems odd.